stdlib/tkinter/__init__.pyi

```diff
@@ -352,7 +355,29 @@ class Misc:
     # after_idle is essentially partialmethod(after, "idle")
     def after_idle(self, func: Callable[[Unpack[_Ts]], object], *args: Unpack[_Ts]) -> str: ...
     def after_cancel(self, id: str) -> None: ...
+    if sys.version_info >= (3, 13):
+        def after_info(self, id: str | None = None) -> tuple[str, ...]: ...
+
     def bell(self, displayof: Literal[0] | Misc | None = 0) -> None: ...
+    if sys.version_info >= (3, 13):
+        # Supports options from `_BusyInfo``
+        def tk_busy_cget(self, option: Literal["cursor"]) -> _Cursor: ...
+        busy_cget = tk_busy_cget
+        def tk_busy_configure(self, cnf: Any = None, **kw: Any) -> Any: ...
+        tk_busy_config = tk_busy_configure
+        busy_configure = tk_busy_configure
+        busy_config = tk_busy_configure
+        def tk_busy_current(self, pattern: str | None = None) -> list[Widget]: ...
```

Comment:
Tkinter's `Widget` is misleadingly named. For this type annotation, you want to say "any widget whatsoever", and to spell that, you actually need `Misc`, not `Widget`.

A `Widget` is something you can add inside another widget, i.e. anything except `Tk` and `Toplevel`. But a `Tk` or `Toplevel` can be marked as busy.

```suggestion
        def tk_busy_current(self, pattern: str | None = None) -> list[Misc]: ...
```